ByBit.Net/Clients/SpotApi/v3/BybitClientSpotApiTradingV3.cs

```diff
@@ -152,20 +152,16 @@ public async Task<WebCallResult> CancelMultipleOrderAsync(string symbol, OrderSi
         #region Get user trades
 
         /// <inheritdoc />
-        public async Task<WebCallResult<IEnumerable<BybitSpotUserTrade>>> GetUserTradesAsync(string? symbol = null, long? fromId = null, long? toId = null, int? limit = null, long? receiveWindow = null, CancellationToken ct = default)
+        public async Task<WebCallResult<BybitSpotUserTradeWrapper>> GetUserTradesAsync(string? symbol = null, long? fromId = null, long? toId = null, int? limit = null, long? receiveWindow = null, CancellationToken ct = default)
```

Comment:
I think it would be more user friendly to still return the list instead of this wrapper. The wrapper doesn't add anything for the user other than another layer. So I'd rather still have it with the previous setup with requesting with the wrapper type, but returning the internal list if the call is succesful

Reply:
Should we return Task<IEnumerable<BybitSpotUserTrade>> instead of Task<WebCallResult...? Or it's better to create a new WebCallResult and pass there Wrapper.Data?

Reply:
The second, create a new WebCallResult and use the wrapper.Data.
You can use the original call result and use `result.As<NewType>([data])` to create a new callresult from the original

Reply:
Thank you for recommendation.